src/routes/index.js

```diff
@@ -13,7 +13,7 @@ const AppRoutes = () => {
                 />
                 <Route
                     path="login"
-                    element={<LoginPage/>}
+                    element={<LoginModule/>}
```

Comment:
LoginPage is a correct word. You're redirecting to the Page, not to the module. Inside page you should use modules